pepper/cli.py

```diff
@@ -108,6 +108,21 @@ def parse(self):
             ''')
         )
 
+        self.parser.add_option(
+            '--state-output', dest='state_output', default="full", type="choice",
+            choices = [ "full", "terse", "mixed", "changes", "filter" ],
```

Comment:
```suggestion
            choices = [ "full", "full_id", "terse", "terse_id", "mixed", "mixed_id", "changes", "changes_id", "filter", "filter_id" ],
```
You might want to add the `*_id` variants as well, which allow to show the states' IDs instead of their `name` as primary output key.
See also:
- PR in our own pepper-fork, where we merged your PR: https://github.com/ssc-services/pepper/pull/2
- [SaltStack Highstate Output docs](https://docs.saltproject.io/en/latest/ref/output/all/salt.output.highstate.html#outputter-for-displaying-results-of-state-runs)